src/protocols/aeternity/views/TransferSendModal.vue

```diff
@@ -72,6 +77,7 @@ export default defineComponent({
     const currentRenderedComponent = ref<Component>();
     const currentStep = ref<TransferSendStep>(TRANSFER_SEND_STEPS.form);
     const error = ref(false);
+    const txRaw = ref();
```

Comment:
I don't see any logic that updates this empty reference. Maybe this can be removed?